docs/telemetry/adapters/kafka/index.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: default
+title: "kafka-telegraf"
+date: 2022-03-1 20:03:00 --0600
+categories:
+---
+
+## Introduction
+
+Kafka is a popular message bus system that allows applications to communicate over a pub/sub bus
+as either a publisher or a consumer.  
+A popular method of distributing streaming telemetry is to take the telemetry data and output it
+to a kafka topic so it can be further reacted upon.
+
+This lab will leverage the telegraf container to take streaming telemetry.  
+from two cEOS lab devices from their gNMI interfaces and output the data to a kafka topic.
+
+## Prerequisite
+
+- [containerlab](https://containerlab.dev/)
+- [docker](https://www.docker.com/)
+- [ceos](https://containerlab.dev/manual/kinds/ceos/)
+
+cEOS lab will need to be downloaded from the arista software downloads
+and imported via docker with a tag of 4.29.2F
+
+## Environment
+
+The containerlab file
+<details><summary>Reveal output</summary>
```

Comment:
Just a minor comment, we can probably remove the html in markdown if we enable admonitions. Then we can do the same with the following.

old

```
<details><summary>Reveal output</summary>
<p>

```bash
--8<-- "src/kafka-telegraf/telegraf.conf"
```

</p>
</details>
```

newer

```
??? example "telegraf.conf"

    ```bash
    --8<-- "src/kafka-telegraf/telegraf.conf"
    ```
```